util/src/main/java/org/datacommons/util/LogWrapper.java

```diff
@@ -106,10 +106,6 @@ public void provideStatus(long count, String thing)
   }
 
   public void persistLog(boolean silent) throws InvalidProtocolBufferException, IOException {
-    if (log.getLevelSummaryMap().isEmpty()) {
-      if (!silent) logger.info("Found no warnings or errors!");
```

Comment:
nit: might be useful to log this message `if (log.getLevelSummaryMap().isEmpty() && !silent)`  ?

Reply:
Nm. summaryString() should be good enough.